WalletWasabi.Fluent/ViewModels/Wallets/Buy/BuyViewModel.cs

```diff
@@ -84,9 +86,9 @@ async Task IOrderManager.RemoveOrderAsync(int id)
 		SelectedOrder = _orders.FirstOrDefault();
 	}
 
-	public void Activate(CompositeDisposable disposable)
+	public void Activate()
 	{
-		Task.Run(async () => await InitializeAsync(disposable), _cts.Token);
+		Task.Run(async () => await InitializeAsync(), _cts.Token);
```

Comment:
Lambda expression can be removed:

![image](https://github.com/zkSNACKs/WalletWasabi/assets/52379387/4e96acfa-6498-491c-905f-5e2e73aea44d)

```suggestion
		Task.Run(InitializeAsync, _cts.Token);
```